admin/src/js/controllers/forms-xml.js

```diff
@@ -25,16 +26,18 @@ angular.module('controllers').controller('FormsXmlCtrl',
     };
 
     const uploadFinished = function(err) {
+      $scope.status.uploading = false;
       if (err) {
         $log.error('Upload failed', err);
+        $scope.status.error = true;
```

Comment:
I'm not sure this is the best way to handle these errors: 
This is how it looks when I try to upload something that is not XML: 
![image](https://user-images.githubusercontent.com/35681649/112793182-7d1fc080-906d-11eb-8399-9a92faeff715.png)

it just prints everything to the screen. Same if I try to upload invalid xml: 
![image](https://user-images.githubusercontent.com/35681649/112793225-932d8100-906d-11eb-997d-1bef9bb4a836.png)

I selected a big form and there is looots of scrolling ;)